user-service/src/main/java/rs/edu/raf/banka1/model/Card.java

```diff
@@ -0,0 +1,30 @@
+package rs.edu.raf.banka1.model;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+public class Card {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(length = 16, unique = true)
+    private String cardNumber;
+    private String cardType;
```

Comment:
zasto ostale stvari nisu @Column? Deluje mi da ovo generise samo 3 kolone

Reply:
Mislim da ne, kolko ja znam @column nije obavezno

Reply:
izgeneriše sve, nego sam sam @column stavljao samo tamo gde je trebalo da postavim neka ograničenja, tako da je okej to u bazi